src/vivarium_census_prl_synth_pop/components/population.py

```diff
@@ -321,9 +325,10 @@ def initialize_newborns(self, pop_data: SimulantData) -> None:
 
         # typing
         new_births["household_id"] = new_births["household_id"].astype(int)
-        # todo: Update guardian columns MIC-3595
-        new_births["guardian_1"] = data_values.UNKNOWN_GUARDIAN_IDX
-        new_births["guardian_2"] = data_values.UNKNOWN_GUARDIAN_IDX
+        # Assign guardian_1 to birth mother
+        new_births["guardian_1"] = new_births["parent_id"]
```

Comment:
`parent_id` is definitively the mother?

Reply:
Yes.  That how it was defined in earlier versions before the guardian functionality was known to us.